docker/Dockerfile

```diff
@@ -4,67 +4,55 @@
 ## Copyright (c) 2015-2020 ODC Contributors
 ## SPDX-License-Identifier: Apache-2.0
 ##
-ARG ORG=opendatacube
-ARG V_BASE=3.3.0
 ARG V_PG=12
-FROM ${ORG}/geobase-builder:${V_BASE} as env_builder
+FROM osgeo/gdal:ubuntu-small-latest as env_builder
```

Comment:
I think having a pinned version here is better than using `-latest`

Reply:
Also don't need `as env_builder` as this stage isn't re-used anywhere, is it?

Reply:
Yes and yes.

Although I kind of like getting heads up when there's a new version out that breaks things.